src/js/step.js

```diff
@@ -450,14 +409,13 @@ export class Step extends Evented {
 
     document.body.setAttribute('data-shepherd-step', this.id);
 
-    this.setupPopper();
```

Comment:
Shouldn't we need to set up tippy on show still?

Reply:
Slightly above this line we do:

```js
    if (!this.el) {
      this.setupElements();
    }
```

`this.el` refers to the content of the tooltip. `this.setupElements` creates, that, and it also creates the tippy instance.

Reply:
Right on, but part of my rationale here was to prevent us from potentially calling the `setupPopper` function twice [as part of what `_show()` does](https://github.com/shipshapecode/shepherd/blob/e3e95ec893dfe0e4de505308e482a37c50efd057/src/js/step.js#L453).